Makefile

```diff
@@ -9,9 +9,37 @@ all:
 build: Dockerfile docker-compose.yml
 	docker-compose build vautomator
 
-.PHONY: scan
-scan:
-	docker run -v ${PWD}/results:/app/results -it vautomator:latest ./run.py $(TARGET)
+.PHONY: force-build
```

Comment:
maybe at this point we should actually use a wrapper rather than Makefile just for CLI
otherwise, this works ok :)

Reply:
I like the current approach. What would be an example wrapper be like?

Reply:
if you like it you should keep it

A wrapper would be "vaautod.py" such as 
```
#!/bin/bash
docker run -v ${PWD}/results:/app/results -it vautomator:latest ./run.py $*
```
The difference is that it allows for multiple arguments in one go without doing `make ARGS="arg1 arg2" TARGET="lala.com" scan` (lots to type) or `make arg TARGET="lala.com"` (2nd one can't take more args)